scripts/OCR_Scanner/ocr.py

```diff
@@ -0,0 +1,16 @@
+from PIL import Image
+import pytesseract
+import numpy as np
+
+#Input your Local Machine Address in filename where you have stored your image.
+filename = 'https://github.com/Harshil-Jani/General-Purpose-Scripts/blob/OCR-Script/scripts/OCR_Scanner/image_01.png'
```

Comment:
Make an input field allowing the user to choose a valid file instead of having to edit the script.